server/src/child.rs

```diff
@@ -331,13 +331,14 @@ fn trace_function(
                         if let Ok(_) = child.read_memory(address, &mut buffer) {
                             let name = symbol.name.to_string_lossy().into_owned();
 
-                            let value = match size {
-                                4 => unsafe { *(buffer.as_ptr() as *const u32) as u64 },
-                                8 => unsafe { *(buffer.as_ptr() as *const u64) as u64 },
-                                _ => unsafe { *(buffer.as_ptr() as *const u32) as u64 },
-                            };
+                            let value = debug::Value::read(&child, &symbols, &context, &symbol);
+                            if let Ok(value) = value {
+                                if value.data[0] == 0xcc {
```

Comment:
Windows' debug-mode allocator initializes memory to `0xcc` to make uninitialized variables stand out in a debugger. I added this to the prototype to hide said uninitialized variables and decided to move it here.